vocabularies/json-ld/patronTypes.json

```diff
@@ -165,8 +175,8 @@
       "@id": "ptype:106",
       "@type": "nypl:PatronType",
       "nypl:deliveryLocationAccess": [
-        "Research",
-        "Scholar"
+        "Scholar",
```

Comment:
I _assume_ this makes no difference... but I'm intrigued my this reversing of elements is the reverse of the reversing of elements in 110. Is there any reason? Not asking for a change just trying to understand!

Reply:
Ah, right — figures. But annoying! But only a bit.